docs/website/docs/user-guides/advanced/debugging-with-openshift-toolkit.md

```diff
@@ -0,0 +1,280 @@
+---
+title: Debugging an Application with OpenShift Toolkit
+sidebar_position: 7
+---
+
+Debugging is an unavoidable part of development, and it can prove even more difficult when developing an application that runs remotely.
+
+However, this task is made absurdly simple with the help of OpenShift Toolkit.
+
+## OpenShift Toolkit
+[OpenShift Toolkit](https://github.com/redhat-developer/intellij-openshift-connector) is an IDE plugin that allows you to do all things that `odo` does, i.e. create, test, debug and deploy cloud-native applications on a cloud-native environment in simple steps.
```

Comment:
The link points to the IntelliJ plugin, but there is no pointer to the one for VS Code. I'd suggest removing this link here, and instead mentioning that there are plugins for VSCode and JetBrains IDEs, with dedicated links to their marketplaces. What do you think?